TensorFlow-Object-Detection.ipynb

```diff
@@ -54,20 +111,75 @@
    "id": "c9784d5a-f1da-4618-9f3a-5c78eae00cd5",
    "metadata": {},
    "source": [
-    "## Download pre-trained model and train your own custom model"
+    "## Download pre-trained model\n",
+    "\n",
+    "You can either download from the Website or directly from the API."
    ]
   },
   {
    "cell_type": "markdown",
    "id": "afc9ce2b-7abf-44f0-8b37-9c637f6de021",
    "metadata": {},
    "source": [
-    "### Download pre-trained model\n",
+    "### Download from AutoML Station's Marketplace Website\n",
     "\n",
     "- Download `SSD MobileNet V2 320x320 - COCO 2017` from ***[AutoML Station's Marketplace](https://marketplace.amlstation.com/models/6)***;\n",
     "- Place `mobilenet_ssd.zip` inside `tensorflow-object-detection/models` and unzip it;"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "253ee105-1fc0-4d73-a14b-cb102e13973e",
+   "metadata": {},
+   "source": [
+    "### Download from AutoML Station's Marketplace API\n",
+    "\n",
+    "Paste your API token to the `API_TOKEN` variable on the cell below.\n",
+    "\n",
+    "You can find instructions for generating a valid API Token on the [README.API.md](./README.API.md) file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "14be6022-2ee2-4cff-aae5-9dca34e42568",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%cd /workspace\n",
+    "\n",
+    "!pip3 install requests --quiet\n",
+    "import requests\n",
+    "\n",
+    "API_TOKEN=\"your-api-token-here\"\n",
```

Comment:
We could remove this line since the API_TOKEN is already being defined on a previous cell. This way the user does not need to edit two different cells in order to use the Notebook.

Reply:
Same applies to:

```
!pip3 install requests --quiet
import requests

API_TOKEN="your-api-token-here"

headers = {
    'Authorization': f"Bearer {API_TOKEN}",
}
```

Reply:
Note: Add a new cell at the beginning of the notebook to configure the API